app/src/main/java/org/imozerov/streetartview/AppComponent.java

```diff
@@ -1,7 +1,7 @@
 package org.imozerov.streetartview;
 
 import org.imozerov.streetartview.ui.detail.DetailArtObjectActivity;
-import org.imozerov.streetartview.ui.explore.list.ArtListFragment;
+import org.imozerov.streetartview.ui.explore.ArtListPresenter;
```

Comment:
Because it will be used for map fragment also. 
It was mu initial intent to encapsulate dataStorage interactions in some object that can be reused for map fragment. (I think that map fragment should have the same functionality as list.